web/build.gradle

```diff
@@ -3,12 +3,22 @@ plugins {
 }
 
 dependencies {
+    constraints {
```

Comment:
Let's look at this in the next PR concerning SAML integration with IdP. We should be careful to add specific versions in submodules.